openfisca_survey_manager/scenarios.py

```diff
@@ -1100,10 +1131,10 @@ def summarize_variable(self, variable = None, use_baseline = False, weighted = F
             if holder.variable.definition_period == ETERNITY:
                 array = holder.get_array(ETERNITY)
                 print("permanent: mean = {}, min = {}, max = {}, median = {}, default = {:.1%}".format(
-                    array.mean() if not weighted else np.average(array, weights = weights),
+                    array.astype(float).mean() if not weighted else np.average(array, weights = weights),
```

Comment:
@claireleroy : `astype(float)` is needed for large integer values. Should we use it for  `np.average` too ?